python/foglamp/plugins/south/systeminfo/systeminfo.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for System Info async plugin """
+
+import asyncio
+import copy
+import uuid
+import datetime
+import os
+import glob
+import sys
+import shutil
+import json
+import fnmatch
+import subprocess
+from foglamp.services.core.connect import *
+from foglamp.common import logger
+from foglamp.plugins.common import utils
+from foglamp.services.south import exceptions
+from foglamp.services.south.ingest import Ingest
+
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2018 Dianomic Systems"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_DEFAULT_CONFIG = {
+    'plugin': {
+        'description': 'System info async plugin',
+        'type': 'string',
+        'default': 'systeminfo'
+    },
+    'assetCode': {
+        'description': 'Asset Code',
+        'type': 'string',
+        'default': "system"
+    },
+    'sleepInterval': {
+        'description': 'Sleep Interval, in seconds, between two System info gathering',
+        'type': 'integer',
+        'default': "30"
+    }
+}
+_LOGGER = logger.setup(__name__, level=20)
+
+
+def plugin_info():
+    """ Returns information about the plugin.
+    Args:
+    Returns:
+        dict: plugin information
+    Raises:
+    """
+
+    return {
+        'name': 'System Info plugin',
+        'version': '1.0',
```

Comment:
should match as you mentioned in VERSION file to make things consistent and meaningful